pandas/tools/tile.py

```diff
@@ -151,6 +151,9 @@ def qcut(x, q, labels=None, retbins=False, precision=3):
         as a scalar.
     precision : int
         The precision at which to store and display the bins labels
+    duplicates : {'raise', 'drop'}, optional
```

Comment:
or you can also add it in the description like "Default is to raise an error" (one of both is good)